tests/Fixer/Basic/NumericLiteralSeparatorFixerTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Basic;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Marvin Heilemann <11534760+muuvmuuv@users.noreply.github.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\Basic\NumericLiteralSeparatorFixer
+ */
+final class NumericLiteralSeparatorFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null): void
```

Comment:
```suggestion
    /**
     * @dataProvider provideFixCases
     */
    public function testFix(string $expected, ?string $input = null): void
```

Reply:
Copied it from here, should I update it there as well? https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/cookbook_fixers.rst

Reply:
sounds good ;)
consider to raise it as separated PR, we can merge docs update earlier.

Reply:
It will be added back after running the fixer